fineract-provider/src/main/resources/sql/migrations/list_db/V1__mifos-platform-shared-tenants.sql

```diff
@@ -67,6 +67,11 @@ CREATE TABLE `tenants` (
 LOCK TABLES `tenants` WRITE;
 /*!40000 ALTER TABLE `tenants` DISABLE KEYS */;
 INSERT INTO `tenants` VALUES (1,'default','Default Demo Tenant','mifostenant-default','Asia/Kolkata',NULL,NULL,NULL,NULL,'localhost','3306','root','mysql',1);
+-- Add tenants to support interoperation multi-tenancy
+-- INSERT INTO `tenants` VALUES (2,'tn01','Buffalo','tn01','Africa/Bujumbura',NULL,NULL,NULL,NULL,'localhost','3306','root','mysql',1);
+-- INSERT INTO `tenants` VALUES (3,'tn02','Lion','tn02','Africa/Bujumbura',NULL,NULL,NULL,NULL,'localhost','3306','root','mysql',1);
+-- INSERT INTO `tenants` VALUES (4,'tn03','Rhino','tn03','Africa/Bujumbura',NULL,NULL,NULL,NULL,'localhost','3306','root','mysql',1);
+-- INSERT INTO `tenants` VALUES (5,'tn04','Elephant','tn04','Africa/Bujumbura',NULL,NULL,NULL,NULL,'localhost','3306','root','mysql',1);
 /*!40000 ALTER TABLE `tenants` ENABLE KEYS */;
```

Comment:
Just a note for future reference that we shouldn't be modifying existing SQL migrations. 
This particular change, however, is not a blocker for merging the PR as you have commented out the changes made and the committer merging the pr would likely undo changes to this particular file before merging the PR